src/main/java/nextstep/subway/path/application/PathService.java

```diff
@@ -0,0 +1,62 @@
+package nextstep.subway.path.application;
+
+import nextstep.subway.line.domain.LineRepository;
+import nextstep.subway.line.domain.Lines;
+import nextstep.subway.path.domain.PathFinder;
+import nextstep.subway.path.dto.PathResponse;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
+import nextstep.subway.station.domain.Stations;
+import nextstep.subway.station.dto.StationResponse;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+@Service
+public class PathService {
+
+    private final LineRepository lineRepository;
+    private final StationRepository stationRepository;
+
+    public PathService(LineRepository lineRepository, StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    public PathResponse findPath(Long sourceStationId, Long targetStationId) {
+        PathFinder pathFinder = initPathFinder();
+
+        validation(sourceStationId, targetStationId);
+        Station sourceStation = findStation(sourceStationId);
+        Station targetStation = findStation(targetStationId);
+
+        List<StationResponse> stationResponses = pathFinder.getPath(sourceStation, targetStation)
+                .stream()
+                .map(station -> StationResponse.of(station))
+                .collect(Collectors.toList());
+
+        PathResponse response = new PathResponse(stationResponses, pathFinder.getWeight(sourceStation, targetStation));
+
+        return response;
```

Comment:
별도의 다른 처리가 없으니, 바로 리턴해주셔도 괜찮지 않을까요?

```suggestion
        PathResponse response = new PathResponse(stationResponses, pathFinder.getWeight(sourceStation, targetStation));

        return new PathResponse(stationResponses, pathFinder.getWeight(sourceStation, targetStation));
```

Reply:
바로 반환하는것으로 변경하였습니다 ^^